src/scripts.js

```diff
@@ -36,22 +36,23 @@ function getRandomUser(array) {
 
 function getUser() {
   loadUsers().then(usersData => {
-    const userData = getRandomUser(usersData);
-    user = new User(userData);
-    getRecipes();
+    getRecipes(usersData);
   });
 }
 
-function getRecipes() {
+function getRecipes(usersData) {
```

Comment:
To avoid passing in usersData along these functions, could we call this getRecipe function first at our event listener, then moves on to getIngredients, and then moves into the getUser?